chromadb/__init__.py

```diff
@@ -181,6 +181,53 @@ def HttpClient(
     return ClientCreator(tenant=tenant, database=database, settings=settings)
 
 
+def CloudClient(
+    tenant: str,
+    database: str,
```

Comment:
I'm getting db-scoped API tokens over the finish line at which point this should become an optional parameter

Reply:
Let's see which one lands first, then this can be modified at that point. Some API tokens will presumably still be at other scopes, where the user will need to know which tenant and which DB they want the client to connect to.